docs/tutorials/t-09.md

```diff
@@ -0,0 +1,123 @@
+# Algo Builder Web Support for Multisignature Transaction
+
+This tutorial demonstrates the usage of Algo Builder Web Package to sign and send a Multisignature Transaction to blockchain network using major Algorand wallets (Algo Signer, Wallet Connect, MyAlgoWallet).
+
+### Requirements
+
+- [Algo Builder](https://algobuilder.dev/) and it's [dependencies](https://algobuilder.dev/guide/README#requirements)
+- Setup the accounts on [`Algosigner`](https://github.com/PureStake/algosigner), [`Wallet Connect`](https://walletconnect.com/) wallet and [`MyAlgo Wallet`](https://wallet.myalgo.com/). It could be MainNet, TestNet or private net.
+
+## Steps
+
+In this tutorial we will use Algo Builder Wallet Widget to connect to various web wallets. Then we will show how to instrument the wallet to sign a multisignature transaction.
+See [Wallet Widget tutorial](https://developer.algorand.org/solutions/vuejs-multi-wallet-widget/) for more details about the widget.
+
+### MultiSignature Transaction
+
+To create a msig transaction, you can follow steps from the offical documentation: [Create Multisignature](https://developer.algorand.org/docs/get-details/accounts/create/#multisignature) and [Transaction](https://developer.algorand.org/docs/sdks/javascript/) to use it.
+Here is a glimpse of creating the multisig transaction:
+
+```js
+const mparams = {
+	version: 1,
+	threshold: 2,
+	addrs: [account1.addr, account2.addr, account3.addr],
+};
+
+let multsigaddr = algosdk.multisigAddress(mparams);
+let msigTxn = algosdk.makePaymentTxnWithSuggestedParamsFromObject({
+	from: multsigaddr,
+	to: receiver,
+	amount: amount,
+	note: note,
+	suggestedParams: params,
+});
+```
+
+After creating the transaction, you can sign it using different wallets.
+
+1. AlgoSigner<br/>
+   We use `connectAlgoSigner()` method to connect the AlgoSigner wallet. Next, we create an instance of the `WebMode` class by passing `AlgoSigner` and the Chain Name i.e the network name. The `web` instance will allow us to sign and send transactions to the blockchain.
+
+```js
+const web = new WebMode(AlgoSigner, CHAIN_NAME);
+```
+
+WebMode method `signMsigTxn` expects txn in [`EncodedSignedTransaction`](https://algorand.github.io/js-algorand-sdk/interfaces/EncodedSignedTransaction.html) type. To convert the above created msig transaction, below code is used:
+
+```js
+let subsig = [];
+for (const address of mparams.addrs) {
+	subsig.push({ pk: decodeAddress(address).publicKey });
+}
+const msigEncodedTxn = {
+	msig: { v: mparams.version, thr: mparams.threshold, subsig: subsig },
+	txn: msigTxn.get_obj_for_encoding(),
+};
+```
+
+To sign the msig transaction:
+
+```js
+const signedTxnResponse = await web.signMsigTxn([msigEncodedTxn]);
+```
```

Comment:
is this going to sign with all available accounts, or only with the account currently seclected in algosigner? What if the selected account is not part of the multisig? Let's clarify this in the description here.